samples/start-app/js-node-express/client/https-client.js

```diff
@@ -26,12 +24,18 @@ class HttpsClient{
 
         const merged_options = {...this.options, ...options}
 
-        console.log("END : " + this.options)
-        console.log(merged_options);
-
         return https.request(merged_options, callback);
     }
 
+    setUserAgent(app_version = undefined, docker_version = undefined) {
+
+        let ret = 'AkeneoSampleApp/js-node-express';
+        ret += (app_version)? ' Version/' + app_version : '';
+        ret += (docker_version)? ' Docker/' + docker_version : '';
+
+        return ret;
+    }
+
 }
 
 module.exports = HttpsClient;
```

Comment:
why not call it `httpClient` ?

Reply:
Because in node you have two internal libraries : http and https. The first one can not process https request. So we don't want to mix up things here as our calls are all ssl secured.